hw/dv/tools/dvsim/vcs.hjson

```diff
@@ -6,6 +6,13 @@
   build_ex:   "{build_dir}/simv"
   run_cmd:    "{job_prefix} {build_ex}"
 
+  // This option enables the following: (needed for uvm_hdl_*)
+  //  - Read capability on registers, variables, and nets
+  //  - Write (deposit) capability on registers and variables
+  //  - Force capability on registers, variables, and nets
+  // TODO Trim this flag since +f hurts performance.
+  vcs_build_opt_debug_access: "-debug_access+f"
```

Comment:
Could we change the default to avoid +f?

Reply:
No, if we do so, those tests that use `uvm_hdl_force` will fail